rplugin/python3/denite/source/defx/bookmarks.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+
+sys.path.append(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+)
+
+if True:
+    import dirmark.util as dm
+    from denite.source.base import Base
+
+
+class Source(Base):
+    def __init__(self, vim):
+        super().__init__(vim)
+
+        self.name = 'defx/bookmarks'
```

Comment:
I use `dirmark` for this plugin's name (because can mark only directory .. ), so want to use it here too.

Reply:
Changed

Reply:
Thanks :)